VERSION

```diff
@@ -1 +1 @@
-0.1.10
+0.1.11
```

Comment:
Not necessarily, version `0.y.z` is for initial development and anything may change at any time, the API is considered unstable. See http://semver.org/ item "4.".